src/rules/declaration-block-trailing-semicolon/__tests__/index.js

```diff
@@ -0,0 +1,153 @@
+import test from "tape"
+import testRule from "../../__tests__/utils/testRule"
+
+import trailingSemicolon from ".."
+
+const testTrailingSemicolon = testRule(trailingSemicolon)
+const expectedError = "Expected a trailing semicolon (declaration-block-trailing-semicolon)"
+const rejectedError = "Expected no trailing semicolon (declaration-block-trailing-semicolon)"
```

Comment:
maybe we can get those messages exported from the rule directly. With es6 we can have multiples values exported: a default one `import rule from "./rule"` and some others like `import rule, {messages} from "./rule"`. That will make maintenance easier. 
In the rule, we can just add `export const messages = { /*...*/ }`